src/ucp/core/ucp_worker.c

```diff
@@ -887,6 +887,21 @@ static int ucp_worker_iface_support_keepalive(ucp_worker_iface_t *wiface)
                               UCT_IFACE_FLAG_EP_KEEPALIVE);
 }
 
+/* Compare 'compare_attr' atomic capabilities to 'test_attr',
+ * and decide if it contains all of 'test_attr' capabilities */
+static int ucp_worker_iface_test_atomic_caps(uct_iface_attr_t *compare_attr,
+                                             uct_iface_attr_t *test_attr)
+{
+    return (ucs_test_all_flags(compare_attr->cap.atomic32.op_flags,
```

Comment:
extra ( ) - in every call to ucs_test_all_flags

Reply:
they are already wrapped with extra ().
maybe another () around the whole `return`?

Reply:
I think Yossi meant that these () are not needed there and can be removed.